docs/sites.yml

```diff
@@ -3391,3 +3391,18 @@
   built_by: Henry Kehlmann
   built_by_url: https://github.com/madhenry/
   featured: false
+- title: DexWallet - The only Wallet you need by Dexlab
+  main_url: "https://www.dexwallet.io/"
+  url: "https://www.dexwallet.io/"
+  source_url: "https://github.com/dexlab-io/DexWallet-website"
+  featured: false
+  description: >-
+    DexWallet is a secure, multi-chain, mobile wallet with an upcoming one-click exchange for mobile.
+  categories:
+    - Blockchain
+    - App
+    - Open Source
+    - React Native
+    - Featured
```

Comment:
Please remove this — we'll periodically rotated featured sites but you shouldn't PR your site in with it featured.